src/test/resources/org/prebid/server/it/gdpr-vendorlist1/79.json

```diff
@@ -645,20 +645,6 @@
         3
       ]
     },
-    {
```

Comment:
This file should not be changed, this specific version vendor-list, created not by our organisation